### DISCUSSION TITLE
Custom Login

### DISCUSSION THREAD

**Discussion by StilauGamer** [coder/code-server#5128] [Q&A]
So I am currently working on my own website where I have a dashboard where I code stuff that I need.

And I would really like to implement the code-server to the login system I have on my website.
I currently have a PHP login system that uses sessions to run.

Is there any way where I could take and disable the current login on the code-server and implement the session part into the main page?
So if ur not logged in through the main dashboard you cannot access the code-server and ur prompted to the login page.

**Comment by lg-nineoneninethreenine** [comment]
I think people say that you should use Pomerium for this. @bpmct whattdya think?

**Reply by StilauGamer** [reply]
I have heard people talk about Pomerium.
What I am mostly looking for is that I can have users connect to the dashboard and if they have permission for the code-server they would be able to go onto it.

The users would be added to a database.

**Reply by lg-nineoneninethreenine** [reply]
Hmm idk brb

**Reply by jsjoeio** [reply]
Pomerium is what comes to mind for me as well. I'm not sure how you would do this otherwise 🤔 You can disable auth with code-server via the `config.yaml` or the CLI flag: `--auth=none`
